File: BaiDuVIPHook.py
import frida,sys

jscode = """

if(Java.available){
   Java.perform(function () {
       var a = Java.use("com.baidu.netdisk.account.io._.a");
       a._.implementation=function (a,b) {
          console.log("Hook Start...");
        send('arg0: '+ a);
        send('arg1: '+ b);
         send('result: '+ this._(a,b));
         return 2;
       }
   });
}

"""
process = frida.get_usb_device().attach('com.baidu.netdisk')
print('[*] process')
script = process.create_script(jscode)
def on_message(message,data):
	print (message)
script.on("message",on_message)

script.load()
sys.stdin.read()

File: BaiDuVIPHook.js
// Java.perform(function () {
//   // Function to hook is defined here
//   var a = Java.use('com.baidu.netdisk.account.io._.a');
//   a._.implementation=function(){
//             return 2;
//         }
// });
//
// if(Java.available){
//     Java.perform(function(){
//         var MainActivity = Java.use("com.luoyesiqiu.crackme.MainActivity");
//         MainActivity.isExcellent.overload("int","int").implementation=function(chinese,math){
//             return this.isExcellent(95,96);
//         }
//     });
//
// }

if(Java.available){
   Java.perform(function () {
       var a = Java.use("com.baidu.netdisk.account.io._.a");
       a._.implementation=function () {
           print('[*] Running CTF');
            console.log('[*] Running CTF ');
           return 2;
       }
   });
}